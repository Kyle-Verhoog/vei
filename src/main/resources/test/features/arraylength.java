public class Arraylength {
  public Arraylength() {}
  public int m() {
    int[] x = new int[42];
    return x.length;
  }
}