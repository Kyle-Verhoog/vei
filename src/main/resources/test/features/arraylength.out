PUBLIC ┃ public
CLASS ┃ class
IDENTIFIER ┃ A
{ ┃ {
PUBLIC ┃ public
IDENTIFIER ┃ A
( ┃ (
) ┃ )
{ ┃ {
} ┃ }
PUBLIC ┃ public
INT ┃ int
IDENTIFIER ┃ m
( ┃ (
) ┃ )
{ ┃ {
INT ┃ int
[ ┃ [
] ┃ ]
IDENTIFIER ┃ x
= ┃ =
NEW ┃ new
INT ┃ int
[ ┃ [
INTEGER_LITERAL ┃ 42
] ┃ ]
; ┃ ;
RETURN ┃ return
IDENTIFIER ┃ x
. ┃ .
IDENTIFIER ┃ length
; ┃ ;
} ┃ }
} ┃ }
